### PR TITLE
9.2.0 release notes and roadmap update

### PR DESCRIPTION
**Pull request by tristantarrant**

### CODE REVIEW

File: release-notes.html.haml [hunk 1]
```diff
@@ -2,6 +2,69 @@
...
+
+.row
+  .col-md-6
+    %i.enormous-icon.fa-th-large.pull-left.icon-2x
+    %h3 Core
+    %ul
+      %li Further improvements to conflict resolution and merge policies
+      %li Reactive streams for increased throughput
+      %li Biased reads for scattered caches
+      %li More accurate off-heap memory calculations
```
Comment (wburns) [inline release-notes.html.haml]: We also have the exception based eviction policy now as well.

File: roadmap.html.haml [hunk 2]
```diff
@@ -12,41 +12,47 @@ title: Road Map
...
     %i.enormous-icon.icon-question-sign.pull-left.icon-2x
     %h3 Query
     %ul
-      %li Continuous query materialized view, aggregations, indexed support for initial state [9.x]
       %li ProtoBuf 3 support for remote queries
+    %i.enormous-icon.fa-database.pull-left.icon-2x
+    %h3 Persistence
+    %ul
+      %li Striped cache stores for faster iteration
+      %li Async persistence layer
```
Comment (wburns) [inline roadmap.html.haml]: I am not 100% that we will get this in this version. We will have async iteration at least :D

File: release-notes.html.haml [hunk 3]
```diff
@@ -2,6 +2,69 @@
...
+    %i.enormous-icon.icon-wrench.pull-left.icon-2x
+    %h3 API
+    %ul
+      %li Multimap caches for embedded and Hot Rod
+      %li Clustered Counters for Hot Rod
+      %li Clustered Locks
+      %li Wildcard configurations
+      %li Cluster-wide cache admin via embedded and remote
```
Comment (galderz) [inline release-notes.html.haml]: * Added CacheStream.collect(Supplier) for more user-friendly clustered collect functionality

File: release-notes.html.haml [hunk 4]
```diff
@@ -2,6 +2,69 @@
...
+    Infinispan 9.2 Release Notes
+    %span{:style => "font-size: 0.4em"}
+
+.row
+  .col-md-6
+    %i.enormous-icon.fa-th-large.pull-left.icon-2x
+    %h3 Core
+    %ul
+      %li Further improvements to conflict resolution and merge policies
+      %li Reactive streams for increased throughput
```
Comment (wburns) [inline release-notes.html.haml]: This also reduces resource utilization (threads)

Comment (wburns) [reply release-notes.html.haml]: Also this only affects the `iterator` & `spliterator` methods.

**Pull request merged**